numpy/polynomial/legendre.py

```diff
@@ -1405,6 +1436,13 @@ def legfit(x, y, deg, rcond=None, full=False, w=None):
 
     Examples
     --------
+    >>> from numpy.polynomial import legendre as L
+    >>> c = (1,2,3)
+    >>> x = numpy.linspace(-10, 10)
+    >>> err = numpy.random.randn(len(x))/10
```

Comment:
CI failure is:
```
File "build/testenv/lib/python3.6/site-packages/numpy/polynomial/legendre.py", line 1441, in Legendre._fit
Failed example:
    x = numpy.linspace(-10, 10)
Exception raised:
    Traceback (most recent call last):
      File "/Users/vsts/hostedtoolcache/Python/3.6.9/x64/lib/python3.6/doctest.py", line 1330, in __run
        compileflags, 1), test.globs)
      File "<doctest Legendre._fit[4]>", line 1, in <module>
        x = numpy.linspace(-10, 10)
    NameError: name 'numpy' is not defined
```

Use `np` instead of `numpy`